metabase/metabase
# List metrics in native editor data reference

Closes https://github.com/metabase/metabase/issues/48190

### Description

Adds a section with all the metrics defined on the table to the Data Reference sidebar in the native query editor.

### How to verify

1. Define a metric on the Orders table
2. New -> SQL Query
3. Open the data reference by clicking on the small book incon
4. Click the ORDERS table
5. There should be a section listing your metric

### Demo

<img width="346" alt="Screenshot 2024-09-30 at 13 56 42" src="https://github.com/user-attachments/assets/fca2275e-11c7-4de7-9f18-7238899754f9">



### Checklist
